bin/kano-ui-autostart

```diff
@@ -127,16 +122,6 @@ kano-tracker-ctl generate hw-info &
 
 # User environment preparations: mouse speed, keyboard, DPMS, tracking, accelerated x11compositor.
 
-# TODO: move me to a systemd unit service
-# Are we in Noobs ? (check for more than 2 partitions)
-# If in noobs, make an xrefresh happen after 50 seconds
-if [ $( lsblk -n | wc -l ) -gt 3 ] ; then
-    (sleep  50; xrefresh ) &
-fi
-
-# if xrefresh is still waiting to run, stop it
-kill %?xrefresh
```

Comment:
Apparently this was to prevent a blank screen on the NOOBS image, let's check if we still need it and if we do we can figure out a better place for it, otherwise 🔫